src/Features/CloudShadows.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "Buffer.h"
+#include "Feature.h"
+
+struct CloudShadows : Feature
+{
+	static CloudShadows* GetSingleton()
+	{
+		static CloudShadows singleton;
+		return &singleton;
+	}
+
+	virtual inline std::string GetName() { return "Cloud Shadows"; }
+	virtual inline std::string GetShortName() { return "CloudShadows"; }
+	virtual inline std::string_view GetShaderDefineName() { return "CLOUD_SHADOWS"; }
+	virtual inline bool HasShaderDefine(RE::BSShader::Type) { return true; }
```

Comment:
Should this really always return true? This should only be true if it supports that shader

Reply:
A la `DynamicCubemaps`, which is also all true. In the future everything under the sun will be affected so I am leaving it true.